fabric-item-api-v1/src/main/java/net/fabricmc/fabric/api/item/v1/FabricItemSettings.java

```diff
@@ -42,6 +42,14 @@ public FabricItemSettings equipmentSlot(EquipmentSlotProvider equipmentSlotProvi
 		return this;
 	}
 
+	/**
+	 * Sets the custom damage handler of the item.
+	 */
+	public FabricItemSettings customDamage(CustomDamageHandler handler) {
```

Comment:
Is this called on items that are not damageable? If this is not called on that case, then you need to add javadoc saying the item must be damagable